changes/issue4198.yaml

```diff
@@ -0,0 +1,20 @@
+# An example changelog entry
+#
+# 1. Choose one (or more if a PR encompasses multiple changes) of the following headers:
+#   - feature
+#   - enhancement
+#   - task
+#   - fix
+#   - deprecation
+#   - breaking (for breaking changes)
+#
+# 2. Fill in one (or more) bullet points under the heading, describing the change.
+#    Markdown syntax may be used.
+#
+# 3. If you would like to be credited as helping with this release, add a
+#    contributor section with your name and github username.
+#
+# Here's an example of a PR that adds an enhancement
+
+enhancement:
+  - "Add terminal flow state handler override  - [#4198](https://github.com/PrefectHQ/prefect/issues/4198)"
```

Comment:
just want to note that, I also see other change-logs putting PR# here.

Reply:
Both are accepted patterns. I'm actually curious if we want to consolidate to just using the PR though @cicdw ?

Reply:
If there's an open issue it generally provides more context than the PR so I tend to link to issues if they exist (personal preference), otherwise I use the PR link.  There's no need to pick a unifying rule here.

Reply:
I always use the PR #, but don't think it matters too much. Having something to link the change back to is the important part.

Reply:
Good point, I'll try to get some clarity on which reference is preferred